styles/partials/_sections.scss

```diff
@@ -99,6 +99,27 @@ ul li {
     }
 }
 
+//Results section
```

Comment:
All of this green section I think is redundant to the section above
See: Line 15

Reply:
oops merge conflict gone wrong!